### PR TITLE
Refactor/rollup

### PR DESCRIPTION
**Pull request by KonghaYao**
In this pull request, I have switched the bundling tool in the project to Rollup and performed the necessary bundling and testing. The goal was to ensure that the generated output files remain identical to the previous build.

To achieve this, I had to modify the relevant code sections related to “Fool Webpack”, as it used certain techniques that caused failures during Vite and other tool bundling.

### CODE REVIEW

File: rollup.config.mjs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const packages = fse.readJSONSync("./package.json");
+function createBanner() {
+    var today = format.asString('yyyy-MM-dd', new Date()); // today, formatted as yyyy-MM-dd
+    var version = packages.version;  // module version
+
+    return String(fse.readFileSync('./src/header.js'))
+        .replace('@@date', today)
+        .replace('@@version', version);
+}
+// 防止打包时删除 ts 的类型注解
```
[the remark this answers is not in the document]
Comment (KonghaYao) [reply rollup.config.mjs]: I deleted it in one commit.

**Comment by KonghaYao** [comment]
I want to write some examples about bundlers like Vite, to test their runtime behavior in the browser. I will create an another PR later.

**Comment by josdejong** [comment]
Thanks @KonghaYao , your Rollup replacement works like a charm, this is impressive! It is nice to see that `requireFoolWebpack` has become redundant now 😄. And I see the minified bundles are a bit smaller than before, that's always good.

I think there are no breaking changes, but I'll publish this as a major release because the changed bundle may have introduced subtle changes that for example Webpack applications may be relying on.

I'll also change the version numbers of the devDependencies in package.json to exact numbers, so installing and building `workerpool` anywhere is guaranteed to give the exact same result.

**Comment by josdejong** [comment]
I've now publish `v7.0.0`.

**Pull request merged**